apps/api/src/app/events/e2e/trigger-event.e2e.ts

```diff
@@ -43,6 +43,7 @@ import { DetailEnum } from '@novu/application-generic';
 import { Novu } from '@novu/api';
 import { SubscriberPayloadDto } from '@novu/api/src/models/components/subscriberpayloaddto';
 import { CreateIntegrationRequestDto, TriggerEventResponseDto } from '@novu/api/models/components';
+import { it } from 'mocha';
```

Comment:
This can be removed as mocha globals are already injected in the global namespace.